### DISCUSSION TITLE
Emote priority: Choosing between 7tv, bttv and ffz

### DISCUSSION THREAD

**Discussion by pr0gresiv** [Chatterino/chatterino2#3264] [Ideas]
Hey,

Since 7tv implemented new halloween emotes it would be great to have a feature where u can choose between those emote applications. The problem is, that if a BTTV and (global) 7TV emote have the same name, the BTTV emote is always prioritised. Therefore it is not possible to use the new halloween emotes.  It would be neat to have a feature, where you could change the priorization of these emotes.

I would really appreciate it if theres an opportunity to add this feature in Chatterino

**Comment by Felanbird** [comment]
If you want this to be fixed in the near future you'll probably have to actually open a discussion on the [7tv repo](https://github.com/SevenTV/chatterino7/discussions/new), as we do not yet support 7tv on the main repo, and it would be more in their interest to allow for 7tv priority.

**Reply by pr0gresiv** [reply]
Okay, I opened a discussion on the 7tv repo. Thank you :)
